01_fib/NicholeL/fib.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func abs(args int) (res int) {
```

Comment:
`arg` (singular) or `val` no need for name return, it is discouraged unless you have good reason (see [Effective Go](https://golang.org/doc/effective_go.html#named-results) and [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters)).

Reply:
ok, thanks for juliaogris's reminder